test.js

```diff
@@ -13,18 +13,19 @@ const noop = () => {};
 
 const context = {
   library: {
-    name: '@expo/rudder-node-sdk',
+    name: 'expo-rudder-sdk-node',
     version,
   },
 };
 
 const metadata = { nodeVersion: process.versions.node };
+const host = 'http://localhost';
 const port = 4063;
 
 const createClient = (options) => {
   options = { ...options };
 
-  const client = new Analytics('key', `http://localhost:${port}`, options);
+  const client = new Analytics('key', `${host}:${port}`, options);
```

Comment:
the error message thrown by `fetch-retry` differs from `axios-retry`. updated test case to match